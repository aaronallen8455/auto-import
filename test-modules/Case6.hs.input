module Case6 where

class C a where
  c :: a

main :: IO ()
main = TIO.putStrLn "hi"
