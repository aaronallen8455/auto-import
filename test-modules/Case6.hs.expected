module Case6 where

import qualified Data.Text.IO as TIO

class C a where
  c :: a

main :: IO ()
main = TIO.putStrLn "hi"
